ethereum/contracts/zksync/Verifier.sol

```diff
@@ -66,161 +66,163 @@ contract Verifier is IVerifier {
     uint256 internal constant VK_LOOKUP_TABLE_TYPE_X_SLOT = 0x200 + 0x4c0;
     uint256 internal constant VK_LOOKUP_TABLE_TYPE_Y_SLOT = 0x200 + 0x4e0;
 
+    uint256 internal constant VK_RECURSIVE_FLAG = 0x200 + 0x500;
+
     /*//////////////////////////////////////////////////////////////
                              Proof
     //////////////////////////////////////////////////////////////*/
 
-    uint256 internal constant PROOF_PUBLIC_INPUT = 0x200 + 0x600 + 0x000;
```

Comment:
I am checking with auditor whether they want to see smaller diff. If they are ok, we can leave these changes otherwise will restore.

Reply:
JFYI: they are ok with bigger diff.